exercicios/para-casa/entregas/cintia-macchia/SavingAccount.js

```diff
@@ -0,0 +1,64 @@
+const { BankAccount } = require('./BankAccount');
+
+class SavingAccount extends BankAccount {
+  #qtdWithdrawal = 0; //quantidade de retiradas b.24h
+  #MAX_OF_WITHDRAWAL = 2; //quantd max de retirtadas gratuitas
+  #withdrawalTax = 0.03; //taxa banco 24h
+  incomeRate;
+  incomeDay;
+
+    constructor(client, bank, accountNumber, agencyNumber, incomeRate, incomeDay){
+      super(client, bank, accountNumber, agencyNumber)
+      this.incomeRate = incomeRate,
+      this.incomeDay = incomeDay
+    }
+    
+    generateIncome(currentDay, incomeDay, incomeRate){
```

Comment:
Como `incomeDay` e `incomeRate` são atributos da classe podemos usá-los aqui com o `this`